cluster-api/util/util.go

```diff
@@ -109,4 +110,17 @@ func NewApiClient(configPath string) (*client.ClusterAPIV1Alpha1Client, error) {
 		return nil, err
 	}
 	return c, nil
+}
+
+func NewKubernetesClient(configPath string) (*kubernetes.Clientset, error) {
+	config, err := clientcmd.BuildConfigFromFlags("", configPath)
```

Comment:
It might make sense to add the same check as initApiClient so we can have more friendly error message if the config is invalid.

Reply:
Done.